nevergrad/optimization/test_doc.py

```diff
@@ -45,15 +45,19 @@ def square(x, y=12):
     with futures.ThreadPoolExecutor(max_workers=optimizer.num_workers) as executor:
         recommendation = optimizer.minimize(square, executor=executor, batch_mode=False)
     # DOC_BASE_3
-    optimizer = ng.optimizers.OnePlusOne(parametrization=instrum, budget=10, num_workers=1)
+    import nevergrad as ng
```

Comment:
please add skipped lines before and after the function definition, for readibility

Reply:
Done!

Reply:
Done, thx.